src/Diagnostics.RuntimeHost/Services/ResourceService/SiteService.cs

```diff
@@ -26,13 +27,24 @@ public async Task<StackType> GetApplicationStack(string subscriptionId, string r
             if (string.IsNullOrWhiteSpace(siteName)) throw new ArgumentNullException("siteName");
 
             string queryTemplate =
-                $@"WawsAn_dailyentity 
+                $@"WawsAn_dailyentity
                 | where pdate >= ago(5d) and sitename =~ ""{siteName}"" and sitesubscription =~ ""{subscriptionId}"" and resourcegroup =~ ""{resourceGroup}"" 
                 | where sitestack !contains ""unknown"" and sitestack !contains ""no traffic"" and sitestack  !contains ""undefined""
                 | top 1 by pdate desc
                 | project sitestack";
 
-            DataTable stackTable = await dp.Kusto.ExecuteQuery(queryTemplate, DataProviderConstants.FakeStampForAnalyticsCluster, operationName: "GetApplicationStack");
+            DataTable stackTable = null;
+            
+            try{
+                if (dataProviderContext.Configuration.KustoConfiguration.CloudDomain == KustoDataProviderConfiguration.AzureCloud)
+                {
+                    stackTable = await dp.Kusto.ExecuteQuery(queryTemplate, DataProviderConstants.FakeStampForAnalyticsCluster, operationName: "GetApplicationStack");
+                }
+            }catch(Exception ex){
+                //swallow the exception. Since Mooncake does not have an analytics cluster
```

Comment:
>//swallow the exception. Since Mooncake does not have an analytics cluster [](start = 16, length = 74)

Is this comment outdated as you are already checking for CloudDomain == AzureCloud ?

As Shekhar was suggesting, would also be lovely to have a property added to Context that tells us what cloud we are hitting (if not too much work to do)...